editor/selectors.js

```diff
@@ -3,6 +3,7 @@
  */
 import moment from 'moment';
 import { first, last, get } from 'lodash';
+import { createSelector } from 'reselect';
```

Comment:
@aduth I think there's a problem somewhere in `rememo`. I've replaced the selector by this:

```js 
export const getBlocks = createSelector(
	( state ) => {
		return state.editor.blockOrder.map( ( uid ) => (
			state.editor.blocksByUid[ uid ]
		) );
	},
	state => state.editor.blockOrder,
	state => state.editor.blocksByUid,
);
```

and I'm getting a new Instance even if the state didn't change.
I'll probably keep reselect for now, and feel free to update once fixed.